app/controllers/users_controller.rb

```diff
@@ -8,6 +8,6 @@ def index
   def show
     @user = User.find(params[:id])
     @happy_count = @user.happy_things.count
-    @words_for_wordcloud = WordAggregator.get_aggregated_words(@user, 50)
+    @words_for_wordcloud = WordAggregator.get_aggregated_words(@user, 40)
```

Comment:
40 - 50 judgement. If it was a constant in the `WordAggregator` you'd have only need to change 1 place.